### ISSUE TITLE
Improve Import Export CSV of products

### ISSUE THREAD

**Issue by oallain** [FriendsOfSylius/SyliusImportExportPlugin#200]
Hello,

I created this issue so that we could discuss the evolution of the import / export CSV of products.

## Currently import / export products to the following features:

**Product**

- Details (Code, Name, Description, Enabled, ...) for one Locale
- Active channels
- Price
- Taxonomy (Main taxon and Product taxon)
- Attributes (one column by attribute type)
- Media (one column by media type)

**Variant**

- Name  for one Locale
- Price by Channel

## Current limitations
- Only one locale is use for product datas
- If you want import datas with Attributes values, you must export once before for got file frame
- Imports are slow

What improvements would you like?

**Comment by vasilvestre** [comment]
I also noticed there's additional attributes that aren't exported.  Options are not exported, association are not, all the variants and variant option value aren't. I can work on these feature actually for ~5 days in the next two weeks

**Comment by alexweggs** [comment]
Hi @oallain ,

I think import speed is really important. I create a new Processor for products import, it only has the details part when updating and it take such a long time !

**Comment by oallain** [comment]
Yes, all imports can by faster. I created an issue #191 but i haven't time to work on it yet ;(

**Comment by alexweggs** [comment]
@oallain exports are quite fast with less datas, but a bigger problem here, rest of website is not reachable until export finished :(

**Comment by olimination** [comment]
Hi,
we have started a rough version of an Import of Products with variants and their options, see here: https://github.com/Comvation/SyliusImportExportPlugin/tree/import-export-product-options-and-variants.

It is just a first start, there are still few missing parts like:

- writing/updating tests
- implementing updated export
- updating docs

Maybe you have few feedbacks on the implementation, what should be corrected/improved/changed, etc.

Thanks.

**Comment by oallain** [comment]
Do you think to make a PR here to had it?

**Comment by olimination** [comment]
@oallain I have created a Draft Pull request here #265
